monitoring/dashboard.py

```diff
@@ -50,23 +51,24 @@
 successRate = GaugePanel(
     title="Success rate",
     dataSource="${DS_PROMETHEUS}",
-    calc="lastNotNull",
+    calc="mean",
     format=UNITS.PERCENT_FORMAT,
     min=0,
     max=100,
+    noValue="-",
     targets=[Target(
         expr="\n".join([
             'sum(rate(http_requests_total{namespace="${namespace}", job="${job}", code=~"2.."}[$__rate_interval])) * 100',  # noqa: E501
             "   /",
-            'sum(rate(http_requests_total{namespace="${namespace}", job="${job}"}[$__rate_interval]))',  # noqa: E501
+            'sum(rate(http_requests_total{namespace="${namespace}", job="${job}"}[$__rate_interval]) > 0)',  # noqa: E501
         ]),
-        instant=True,
         legendFormat="Success rate",
     )],
     thresholds=[
-        Threshold("red",    0, 0.0),
-        Threshold("orange", 1, 80.0),
-        Threshold("green",  2, 90.0),
+        Threshold("#808080", 0, 0.0),
+        Threshold("red",     1, 0.0),
```

Comment:
Just to make sure I understand, the index 0 will be used when the value is `null` according to https://github.com/scality/cloudserver/pull/4394/files#diff-636a1e6d6da977cd13d38ab0bce8f93afcf8de9e18aae154dd76588edb205977R254, so why is the percentage set as 0 here?

Reply:
This is the "old" Grafana threshold configuration. First entry (index 0) is the default color, and the "value" is not used.

In this case, we set index 1 from 0.0 : so that grey is used when there is no value (null), ie not any query for some time (or since boot?)

Reply:
Thanks! I understand